css/article_page.css

```diff
@@ -0,0 +1,149 @@
+/* Указываем box sizing */
+*,
+*::before,
```

Comment:
you should place common lines of code to common css file
don't repeat yourself
you didn't resolve this problem